components/centraldashboard/public/components/main-page.css

```diff
@@ -92,16 +92,6 @@ app-toolbar {
     transition: background .25s;
 }
 
-app-toolbar[blue] {
-    color: white;
-    background: var(--primary-background-color);
-    --primary-text-color: white;
-    --secondary-text-color: #ffffff69;
-    --primary-color: white;
-    --accent-color: white;
-    --logo-color: white;
-}
-
```

Comment:
Would we like to leave the support in, in case we ever want the toolbar to change color in the future?